test/cmd/builds.sh

```diff
@@ -15,6 +15,12 @@ project="$(oc project -q)"
 
 oc new-build openshift/ruby-20-centos7 https://github.com/openshift/ruby-hello-world.git
 oc get bc/ruby-hello-world
+# take just a snapshot
+oc logs bc/ruby-hello-world
```

Comment:
This shouldn't work.  test-cmd only starts a master, no nodes, so the pod shouldn't ever be in a running state.